Uchu.StandardScripts/General/StatPickups.cs

```diff
@@ -13,58 +14,76 @@ public override Task LoadAsync()
                 Listen(player.OnLootPickup, lot =>
                 {
                     var stats = player.GetComponent<DestroyableComponent>();
-                    
+                    var skill = player.GetComponent<SkillComponent>();
+                    var missionInventoryComponent = player.GetComponent<MissionInventoryComponent>();
+                    var skillExecute = 0;
```

Comment:
For this, you may find a dictionary lookup (`Dictionary<Lot, int>`) is easier to understand and shorter than this switch statement.